src/irmin-pack/checks.ml

```diff
@@ -193,6 +205,10 @@ struct
       Term.
         (Check_self_contained.term $ setup_log, info ~doc "check-self-contained")
 
+    let rebuild_index =
+      let doc = "Reconstruct index from an existing pack file." in
+      Term.(Rebuild_index.term $ setup_log, info ~doc "rebuild-index")
```

Comment:
This command will replace the existing index with a new one. I'm wondering if we should provide an extra argument to the command, which is the path for the reconstructed index? We often want to compare the old with the one, to check for differences. 
Or maybe the command should first move the existing index into a `index_old` so that it's not overwritten? And the user can then specify whether they want to keep the old index or not.

Reply:
That's a good idea - I've just pushed a commit that adds an optional output argument to specify the destination directory